ls1intum/Artemis
# Bugfix/team exercises/back button not working

### Checklist

### Motivation and Context


Fixes #1895 

### Description

- Set only students of a team to null instead the whole team attribute

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create Team Exercises (text, modeling, file upload and programming)
4. Participate in Team Exercises
5. Assess Team Exercises (make sure you are logged in only as a tutor)
6. Click on back button and check if you are routed back to the team overview

### Screenshots


![video](https://user-images.githubusercontent.com/41108487/89357601-3be33680-d6c1-11ea-9f9e-eca184073793.gif)
